IdentityCore/IdentityCore.xcodeproj/project.pbxproj

```diff
@@ -1714,6 +1734,11 @@
 		1EE42FEE248825CE00899491 /* MSIDAccessTokenWithAuthScheme.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = MSIDAccessTokenWithAuthScheme.m; sourceTree = "<group>"; };
 		1EE5413E2458B30300A86414 /* MSIDDevicePopManager.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = MSIDDevicePopManager.h; sourceTree = "<group>"; };
 		1EE5413F2458B30300A86414 /* MSIDDevicePopManager.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = MSIDDevicePopManager.m; sourceTree = "<group>"; };
+		1EE8FF5624F4BB3800CA1445 /* libIdentityCoreSwift.a */ = {isa = PBXFileReference; explicitFileType = archive.ar; includeInIndex = 0; path = libIdentityCoreSwift.a; sourceTree = BUILT_PRODUCTS_DIR; };
+		1EE8FF5824F4BB3800CA1445 /* IdentityCoreSwift.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = IdentityCoreSwift.swift; sourceTree = "<group>"; };
+		1EE8FF6224F4C0E600CA1445 /* IdentityCoreTests iOS-Bridging-Header.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = "IdentityCoreTests iOS-Bridging-Header.h"; sourceTree = "<group>"; };
+		1EE8FF6324F4C0E600CA1445 /* IdentityCoreTests Mac-Bridging-Header.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = "IdentityCoreTests Mac-Bridging-Header.h"; sourceTree = "<group>"; };
+		1EE8FF6424F4C0E600CA1445 /* File.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = File.swift; sourceTree = "<group>"; };
```

Comment:
nit: can we call it something more representative other than File.Swift :)?

Reply:
Yes this was a dummy file needed like we discussed to generate bridging header. I can maybe rename it to Dummy.swift. :)